influxql/engine.go

```diff
@@ -744,6 +746,54 @@ func ReduceMax(key Key, values []interface{}, e *Emitter) {
 	}
 }
 
+// MapStddev collects the values to pass to the reducer
+func MapStddev(itr Iterator, e *Emitter, tmax int64) {
+	var values []float64
+
+	for k, v := itr.Next(); k != 0; k, v = itr.Next() {
```

Comment:
You should have something here that emits the values after it gets to a certain size. Like say 1000. This may fall over later if you try to emit a values array that is super massive.